Formula/iasql.rb

```diff
@@ -1,7 +1,7 @@
 class Iasql < Formula
   desc "Infrastructure as SQL"
   homepage "https://iasql.com"
-  url "https://github.com/iasql/releases/releases/download/v0.2.2/iasql-macos.tar.gz"
+  url "https://github.com/iasql/cli/releases/download/v0.2.2/iasql-macos.tar.gz"
```

Comment:
This needs to be 0.2.3 right?

Reply:
Oh the version and sha256 get updated when running https://github.com/iasql/homebrew-core/actions/workflows/update-iasql.yml after landing this PR